src/components/markdown/Preview.js

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import marked from 'marked';
 import styles from 'github-markdown-css/github-markdown.css';
+import { useSelector, getMarkdownState } from '../../hooks/appContext';
 
-const Preview = ({ markdown }) => {
+const Preview = () => {
+  const markdown = useSelector(getMarkdownState);
   const __html = marked(markdown);
```

Comment:
add the ability to convert your markdown to a preview to your selectors. `getMarkdownPreview`